src/App.tsx

```diff
@@ -17,18 +54,31 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                setFilter={(value: string) => setFilter(value)}
```

Comment:
Will works same, but less callbacks
```suggestion
                setFilter={setFilter}
```